pkg/operator/deploy/staticresources/master/deployment.yaml.tmpl

```diff
@@ -25,8 +30,13 @@ spec:
         args:
         - operator
         - master
-        image: GENERATED
+        image: "{{ .Image }}{{ if .HasVersion}}:{{ if .IsLocalDevelopment }}{{ .GitCommit }}{{ else }}{{ .Version }}{{end}}{{end}}"
```

Comment:
it looks weird to me to have the colon after the first condition, is this required? The result should look like: `image:gitCommit` or `image:version` so could the colon be moved to after `{{.Image}}`? Same for in the worker file.

Reply:
It was failing the unit tests if not done this way , because if you don't have a version you end up with `image:` without a tag, and then it's not valid

Reply:
I see, thank you!